README.md

```diff
@@ -79,8 +79,8 @@ The request object contains the following properties
 | --------- | ----------- |
 | method    | the HTTP method (GET, POST, PUT, DELETE) |
 | headers   | the HTTP request headers |
-| entityId  | executed when the DELETE command is invoked on a collection |
-| collectionName | the name of the collection |
+| entityId  | executed when the DELETE command is invoked on a serviceObject |
+| serviceObjectName | the name of the serviceObject |
```

Comment:
This property change will require a corresponding change in KCS, which passes the property.